pkg/defaults/defaults.go

```diff
@@ -169,3 +177,32 @@ func processOpSrc(client wrapper.Client, defaultOpsrc v1.OperatorSource) error {
 
 	return nil
 }
+
+func (d *defaults) WaitForDeleteIfDefault(client wrapper.Client, opsrc *v1.OperatorSource) error {
+	logrus.Infof("[defaults] Checking if cleaned up OperatorSource %s is a default", opsrc.Name)
+	_, present := defaultsTracker[opsrc.Name]
```

Comment:
I am adding a helper called `IsDefaultSource()` that can be used. I suggest waiting for #227 to go in and use that.